docs/usage.md

```diff
@@ -5,90 +5,225 @@
+
+Following playbooks can be run to simulate actual actions that can be performed using FortiSOAR&trade; playbooks:
+
+
+- *Isolate Devices From Network*
+- *Scan All Assets Involved*
+- *Update Firewall Policy*.
+
+## OT - Asset Change Activity Scenario
+
+This scenario generates sample *Asset Change Activity* records as follows:
+
+| Type                          | Asset Name          | Record Title                                      |
+|:------------------------------|:--------------------|:--------------------------------------------------|
+| Add Cyber Asset               | Siemens S7-400-S621 | Add Cyber Asset - Siemens S7-400-S621             |
+| Medium Impact Baseline Change | Stratix 5950-RW41   | Medium Impact Baseline Change - Stratix 5950-RW41 |
+
+These sample records demonstrate the process of conducting an *Asset Change Activity* for an asset. The sample Asset Change Activity records have a tag, `Sample`, for easy identification. 
+
+To run this scenario and generate the assets, perform the following steps:
+
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
+
+2. Select the scenario **OT - Asset Change Activity Scenario**.
+
+3. Click **Simulate Scenario**.
+
+    <table>
+        <tr>
+            <td><strong>NOTE</strong></td>
+            <td><strong>Scenario - OT - Asset Change Activity</strong> has a variable <code>dueDays</code> (<em>Days required to complete change activity</em>). You can populate these variables with your preferred values in the <em>Configuration</em> step.</td>
+        </tr>
+    </table>
+
+4. Under the FortiSOAR&trade; menu select to open **Resources** > **Asset Change Activities**.
+
+    ![](./res/asset-change-activities.png)
+
+5. Open any of the activities listed.
+
+    >For documentation purposes, we have opened the **Medium Impact Baseline Change - Stratix 5950-RW4** activity.
+
+The **Assets & Vulnerabilities** tab in the *Medium Impact Baseline Change - Stratix 5950-RW4* activity lists the linked assets.
+
+### Recommended Actions for Asset Change Activity
+
+1. Click the **Add Task From Templates** button to launch **Specify Assignees and Approvers for Tasks** modal.
+
+    ![](./res/asset-change-assign-approve-tasks.png)
+
+2. Fill in the assignees and approver information.
+
+3. Click **Add Task** button to add this task. This creates a new task with priority *Medium* under the **Task Management** tab.
+
+4. Select the **Task Management** tab to open the task management board.
+
+    ![](./res/asset-change-activity-pm-board.png)
+
+5. Edit the open task and mark the status as completed. This moves the task to the *Completed* bucket and creates another task with priority *Medium* under the *Open* bucket. This process repeats until all tasks are moved to *Completed* bucket.
+
+<table>
+    <tr>
+        <td><strong>TIP</strong></td>
+        <td>To add new tasks, click the <strong>Add New Task</strong> button.</td>

```

Comment:
To add standalone new tasks, click.........

Reply:
Not fixed